ldk/javascript/src/whisper/react/component-handlers.ts

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable */
```

Comment:
Did you intend to keep the eslint ignoring the entire file?

Reply:
Nope! Thanks for the catch!